superset-frontend/src/SqlLab/components/SqlEditor/index.jsx

```diff
@@ -545,7 +545,7 @@ const SqlEditor = ({
             />
           </Menu.Item>
         )}
-        {scheduledQueriesConf && (
+        {Object.keys(scheduledQueriesConf).length > 0 && (
```

Comment:
```suggestion
        {Object.keys(scheduledQueriesConf || {}).length > 0 && (
```I'd suggest this In case scheduledQueriesConf is null.